js/main.js

```diff
@@ -0,0 +1,412 @@
+        ) {
+          symbol = symbolChoise(el);
+        }
+      });
+    });
+
+    if (specialBtn === 'tab') {
+      symbol = '  ';
+    }
+
+    if (specialBtn === 'enter') {
+      symbol = '\n';
+    }
+
+    // Adding symbol to textArea
+    textArea.setRangeText(symbol, textArea.selectionStart, textArea.selectionEnd, 'end');
+
+    if (specialBtn === 'backspace') {
+      if (textArea.selectionStart > 0) {
+        const pos = textArea.selectionStart;
+        textArea.value = textArea.value.slice(0, pos - 1) + textArea.value.slice(pos, textArea.value.length);
+        textArea.setRangeText('', pos - 1, pos - 1, 'end');
+      }
+    }
+
+    if (specialBtn === 'del') {
+      const pos = textArea.selectionStart;
+      if (textArea.selectionStart <= textArea.value.length) {
+        textArea.value = textArea.value.slice(0, pos) + textArea.value.slice(pos + 1, textArea.value.length);
+        textArea.setRangeText('', pos, pos, 'end');
+      }
+    }
+
+    const specialBtnEl = document.querySelector(`.${specialBtn}`);
+
+    if (
+      specialBtn === 'shift-left'
+        || specialBtn === 'shift-right'
+        || specialBtn === 'capslock'
+    ) {
+      if (shiftPress === false) {
+        addActiveState(specialBtnEl);
+        caseUp();
+      } else {
+        removeActiveState(specialBtnEl);
+        caseDown();
+      }
+    }
+
+    if (specialBtn === 'arrow') {
+      const pos = textArea.selectionStart;
+
+      switch (targetBtnName) {
+        case 'ArrowUp': setCaretPosition(textArea, pos - 69);
+          break;
+        case 'ArrowRight': setCaretPosition(textArea, pos + 1);
+          break;
+        case 'ArrowDown': setCaretPosition(textArea, pos + 69);
+          break;
+        case 'ArrowLeft':
+          if (textArea.selectionStart > 0) {
+            setCaretPosition(textArea, pos - 1);
+          }
+          break;
+        default:
+          break;
+      }
+    }
+
+    if (specialBtn === 'alt-left') {
+      altLeftPress = activeBtnHighlighting(altLeftPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'alt-right') {
+      altRightPress = activeBtnHighlighting(altRightPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'ctrl-left') {
+      ctrlLeftPress = activeBtnHighlighting(ctrlLeftPress, specialBtnEl);
+    }
+
+    if (specialBtn === 'ctrl-right') {
+      ctrlRightPress = activeBtnHighlighting(ctrlRightPress, specialBtnEl);
+    }
+  }
+
+  textArea.focus();
+}
+
+document.addEventListener('keydown', shiftUpKeyboard);
+keyboard.addEventListener('click', printingInTextArea);
+document.addEventListener('keyup', shiftDownKeyboard);
+document.addEventListener('keydown', (evt) => {
+  if (evt.shiftKey && evt.altKey) pageLangChanging();
+});
+pageLangBtn.addEventListener('click', pageLangChanging);
+
+document.addEventListener('keydown', (evt) => {
+  let symbol = '';
+  textArea.focus();
+
+  keyboardKeys.forEach((row) => {
+    row.forEach((el) => {
+      if (el[1] === evt.code
```

Comment:
```
if(['Backspace', 'Delete', ... , ArrowLeft'].every(code => code !== evt.code)){
        e.preventDefault();
        symbol = symbolChoise(el);
}
```

One small notes.  It would be better to comare string in the same register. Probably it makes sense to call String.toLowerCase of both cases